### DISCUSSION TITLE
Flatpak version

### DISCUSSION THREAD

**Discussion by ghost** [streamdeck-linux-gui/streamdeck-linux-gui#8] [Ideas / Feature Requests]
Flatpak is getting popular. Gnome and KDE are pushing that project so much and Streamdck GUI doesn't want to miss that train.
Flathub already has Streamdeck software called Boatswain but it doesn't work at all so this can be the point where Streamdeck GUI comes to major software for all linux users.

**Comment by coolapso** [comment]
I personally I am not a fan of it, but I do not oppose to if if anyone wants to go for it and submit a PR with the flat-pack build.

**Comment by sftim** [comment]
BTW Boatswain is https://apps.gnome.org/en-GB/app/com.feaneron.Boatswain/

**Reply by nicoaarnio** [reply]
Boatswain doesn't do anything except let you change your streamdeck screen brightness. I don't know why that even exists.
